internal/services/consumer.go

```diff
@@ -15,39 +15,66 @@ import (
 	"github.com/strimzi/strimzi-canary/internal/config"
 )
 
-type Consumer struct {
-	config        *config.CanaryConfig
+type ConsumerService interface {
+	Consume()
+	Refresh()
+	Close()
+}
```

Comment:
Actually, there is not so much gain in this case.
Maybe we are going to add a new Producer for reaching the Kafka cluster from outside the K8S and that interface could be useful; maybe not.
Do you think we should remove them and readding them only in case we find them really useful?

Reply:
IMHO, remove for now and re-add when we need to - I suspect they may come back soon enough around testing, but if we're not sure, I think it's good to remove it.